### DISCUSSION TITLE
Open UPM profile page

### DISCUSSION THREAD

**Discussion by WaynGames** [openupm/openupm#3313] [Ideas]
There is a list of contributors (https://openupm.com/contributors/) that redirect to the git profile of the contributor.
Could we have an open upm profile page that list all the packages owned and discovered by this contributor ?
It would provide an social feature to the site where we could share all our packages with a single url.

**Comment by favoyang** [comment]
I like the idea, but no ETA.

**Comment by favoyang** [comment]
Implementing contributor profile pages in openupm-next via PR https://github.com/openupm/openupm-next/pull/39.

Profile pages are available at `/contributors/<github-user>/` and show separate owned and discovered package sections. Contributor links from the package metadata, package cards, and top hunter/owner lists now point to the OpenUPM profile pages, with a GitHub link kept in the profile header.

**Comment by favoyang** [comment]
Shipped in openupm/openupm-next#41.

Contributor profile pages are now available at `/contributors/<github-user>/`, showing owned and discovered packages in simple tables sorted by latest submission first.
